xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShaderGLES.cpp

```diff
@@ -320,7 +320,9 @@ void YUV2RGBFilterShader::OnCompiledAndLinked()
   glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
 
   GLvoid* data = (GLvoid*)kernel.GetFloatPixels();
-  glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA32F, kernel.GetSize(), 1, 0, GL_RGBA, GL_FLOAT, data);
+#if defined (GL_ES_VERSION_3_0)
+  glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA16F, kernel.GetSize(), 1, 0, GL_RGBA, GL_FLOAT, data);
+#endif
```

Comment:
Isn't here the else path missing now?

Reply:
No. That's just so it compiles. GL_RGBA16f is not available on GLES 2.0. The shader needs 3.1, which is gated here; https://github.com/xbmc/xbmc/blob/21134467d61847e95e32925e0003585d46a715dd/xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGLES.cpp#L1809.

Reply:
... speaking of which, I should probably ifdef the check too, so the scalers are disabled on compile time, if we only GLES 2.0 headers are present.

Reply:
Done.